static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -20,28 +17,50 @@
         text-align: right;
     }
 
-    * + .advert {
-        border-left-color: guss-colour(paid-article-subheader-bg, $frontend-palette);
+    .adverts__row > * + *::before,
+    .adverts__row--wrap::before,
+    .adverts__row--wrap::after {
+        background: guss-colour(paid-article-subheader-bg, $frontend-palette);
+    }
+
+    .adverts__more > summary {
```

Comment:
I don't know. I've always resisted the urge to add class names to everything, especially if the underlying markup is fixed. But this is not a great argument.